content/reference/rest-api/templates/publish.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Templates: publish'
+weight: 10
+
+menu:
+  main:
+    name: 'publish'
+    identifier: 'rest-api-templates-publish'
+    parent: 'rest-api-templates'
+    pre: 'Publishes the current version of a template.'
+---
+
+Publishes the current state of a template as a new version.
+
+# Request
+
+## HTTP Request
+
+```
+POST https://cawemo.com/api/v1/templates/:id/publish
```

Comment:
We could also offer the publishing as an optional parameter for the create and update actions. This way, users would only need to make one call to update and publish a template. 

If API users need a separate publish endpoint, we could add that easily in a future iteration.

Reply:
Agreed, I removed this endpoint and added the `publishVersion` query parameter instead (see https://github.com/camunda/camunda-docs-cawemo/pull/118#discussion_r687661461)

Reply:
See https://github.com/camunda/camunda-docs-cawemo/pull/118#discussion_r688273234